notebooks/visualization/MetPy SkewT.ipynb

```diff
@@ -0,0 +1,298 @@
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from mpl_toolkits.axes_grid1.inset_locator import inset_axes\n",
+    "from metpy.plots import SkewT, Hodograph\n",
+    "from metpy.units import pandas_dataframe_to_unit_arrays, units\n",
+    "import numpy as np\n",
+    "from siphon.simplewebservice.wyoming import WyomingUpperAir\n",
+    "from metpy.calc import wind_speed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Access archived sounding data\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first step in plotting a skew-T is accessing the sounding data. We're going to download some observed sounding data from the University of Wyoming sounding archive from 00z on 15 April 2015 at Topeka, KS during a major tornado outbreak. First, we create a datetime object for MetPy to use to select the correct time and station from the Wyoming archive."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dt = datetime(2012, 4, 15, 0)\n",
+    "station = 'TOP'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll download the data from the Wyoming archive into a Pandas dataframe with units. You'll need an internet connection for this step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = WyomingUpperAir.request_data(dt, station)\n",
+    "data = pandas_dataframe_to_unit_arrays(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll split that Pandas dataframe up into individual arrays for each variable with units, and calculate wind speed from the wind components."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p = data['pressure']\n",
+    "T = data['temperature']\n",
+    "Td = data['dewpoint']\n",
+    "u = data['u_wind']\n",
+    "v = data['v_wind']\n",
+    "spd = wind_speed(u,v)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Plot the Skew-T\n",
+    "\n",
+    "Now we'll plot the data we downloaded on a skew-T diagram.\n",
+    "\n",
+    "The first two lines of code in this cell create a figure using Matplotlib and then modifies it into a skew-T plot\n",
+    "using MetPy's SkewT class.\n",
+    "\n",
+    "The next three lines plot temperature and dewpoint data as red and green lines on the Skew-T, as well as plotting the wind data as barbs along the right side of the plot.\n",
+    "\n",
```

Comment:
This section I think would be more understandable if you break each of these comments into a separate markdown cell and have a few separate code cells as well, like you had done in the first objective.

ie: It will go:

<markdown cell explaining what the code lines mean>
<code cell>
<markdown cell explaining the next few lines>
<code cell>

etc.